README.md

```diff
@@ -1,24 +1,26 @@
-# Frobnicator (this is a template)
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/M0kyOMLZ)
+# Chess 2
 
 ## Authors
-- Andrzej Głuszak (@agluszak on GitHub)
-- Linus Torvalds (@torvalds on GitHub)
+- Jacek Kajdan (@JacekKajdan on GitHub)
 
 ## Description
-Frobnicator is going to be a platformer game similar to Super Mario Bros made using Korge game engine.
+Moim pomysłem jest stworzenie gry desktopowej opierającej się na szachach. Planuję dodać więcej figur o różnych sposobach poruszania się i mocach (przykładowo nekromanta wskrzeszający zbite pionki). Gracz miałby możliwość wyboru figur oraz dowolnego rozstawienia ich na planszy. Chciałbym także aby gracze mieli do dyspozycji plansze o różnych wymiarach.
 
 ## Features
-- map generator
-- shooting
-- enemy AI
-- game state saving and loading
-- scores
+- Wiele nowych figur
+- Możliwość wyboru ustawienia początkowego
+- Plansze o różnych wymiarach
+- Możliwość gry z komputerem
+- Możliwość gry z innym graczem przy jednym komputerze
+### Dodatkowe rzeczy gdyby to co powyżej to było za mało:
+- Możliwość gry na nieskończonej planszy
+- Animacje ruchów specjalnych figur
 
 ## Plan
-In the first part we're going to implement the basics: movement, physics and shooting. The enemies will simply bounce from one edge of the platform to the other. There will be only a single map.
+W pierwszej części planuję stworzyć działającą aplikację w której da się grać w tradycyjne szachy z innym graczem lub z komputerem (silnik szachowy byłby jeszcze bardzo prosty - później i tak sporo się w nim zmieni).
```

Comment:
nie znam się za bardzo na szachach, ale implementacja silnika szachowego to, o ile mi wiadomo, bardzo trudne zadanie. Zgaduję, że będzie Pan bazował na jakichś istniejących implementacjach, np. opisanych tu https://www.chessprogramming.org/Main_Page